openmetadata-docs-v1/content/v1.0.0/connectors/ingestion/workflows/profiler/auto_tagging.md

```diff
@@ -4,15 +4,14 @@ slug: /connectors/ingestion/auto_tagging
 ---
 
 ## Auto PII Tagging
-Here, we are tagging PII Sensitive/NonSensitive tag to column based on the following ways
+Here, we are tagging PII Sensitive/NonSensitive tag to column based on the following ways.
 
-### During Profiler Ingestion
-- During profiler ingestion, we profiler through the sample data.
-- This sample data is passed through a [presidio](https://microsoft.github.io/presidio/)library, wich give spacy entity name, which is used to determine PII Sensitive/NonSensitive Tag.
+PII Tagging is only available during `Profiler Ingestion`.
 
 
-#### Case: Profiler Ingestion 
-Here, if `Auto PII Tagging` is enabled during Metadata Ingestion, we skip the `Auto PII Tag` in Profiler Ingestion even if enabled.
+### During Profiler Ingestion
+- During profiler ingestion, we profile sample data.
+- First we pass the column name through a regex, which identify credit card,email , etc.
```

Comment:
Shouldn't this be before the bullet point above if it is "first"?

Reply:
So here `first` signifies the first method for scanning PII

Reply:
Let's say we have sample data as well. Do we do 1. column scanning and 2. sample data scanning?

Reply:
Yes 👍